<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lab5Disign</class>
 <widget class="QWidget" name="lab5Disign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="plotButton">
   <property name="geometry">
    <rect>
     <x>772</x>
     <y>530</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>440</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Xmin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>480</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Xmax</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="XMinlineEdit">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>442</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="XMaxlineEdit">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>482</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
